modules/default/calendar/calendar.js

```diff
@@ -678,6 +710,7 @@ Module.register("calendar", {
 					// Get the default prefix for this class name and add to the custom symbol provided
 					const className = this.getCalendarProperty(event.url, "symbolClassName", this.config.defaultSymbolClassName);
 					symbols[0] = className + ev.symbol;
+					symbols[0] = ev.symbol;
```

Comment:
this seems buggy, since you overwrite the line before...

Reply:
Yeah, I just pushed a fix